File: rapi/src/main/kotlin/me/gachalyfe/rapi/domain/model/SpecialInterception.kt
package me.gachalyfe.rapi.domain.model

data class SpecialInterception(
    val id: Long,
    val date: String,
    val bossName: String,
    val t9Equipment: Int,
    val modules: Int,
    val t9ManufacturerEquipment: Int,
    val empty: Int,
    val equipments: List<ManufacturerEquipment> = emptyList(),
)

File: gl-frontend/src/models/domain/SpecialInterception.ts
import { ManufacturerEquipment } from './ManufacturerEquipment'

export interface SpecialInterception {
  id?: number
  date: string
  bossName: string
  t9Equipment: number
  modules: number
  t9ManufacturerEquipment: number
  empty: number
  equipments: ManufacturerEquipment[]
}
